docs/RELEASE.md

```diff
@@ -52,3 +52,11 @@ Backups will be established automatically by LTOps and utilized by LF through th
 1. Scale the app up (LTOps/App team)
 1. Test the app (App team)
 1. Communicate maintenance completion
+
+### Database upgrades
+
+Since database upgrades are so infrequent, require extra care and a brief outage, they are to be done manually:
```

Comment:
I would like to make db deployments be automatic as the other deployments already are.

Reply:
I've reverted those changes based on our last conversation